docs/UserWallet.rst

```diff
@@ -58,17 +53,30 @@ A warning message appears behind the label "My wallet"
 
 |image5|
 
+Once your wallet is approved by an administrator, you recieve a notification informing you that your wallet is activated.
+
+|image|
+
+All actions become available and you are allowed to send and recieve funds as well as consulting the transactions hoistory, rewarding details and see the transactions statistics.
+
+It is possible to filter ...
+
+|image|
+
 
 Wallet security
 ~~~~~~~~~~~~~~~~
 
-Each wallet uses a pair of public and private keys. The public key is used ad the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save in case of loss  :
+Backup your wallet
+------------------
+
+Each wallet uses a pair of public and private keys. The public key is used as the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save in case of loss  :
```

Comment:
```suggestion
Each wallet uses a pair of public and private keys. The public key is used as the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save it in case of loss  :
```